### PR TITLE
CI: Add eslint step and prettier step

### PR DESCRIPTION
**Pull request by BenediktSeidl**
Also change to official grafana github actions

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      - name: Cache yarn cache
+        uses: actions/cache@v2
+        id: cache-yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Cache node_modules
```
Comment (kain88-de) [inline .github/workflows/ci.yml:35]: is js dependency management really that broken that two caches are needed?

Comment (BenediktSeidl) [reply .github/workflows/ci.yml:35]: I basically trust the Grafana starter template and hope that it does the right thing.
Perhaps it's some premature optimization, that is not strictly necessary, but speeds up the action for more complicated setups?

There seems to be a shortcut for setup-node that could be used: https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#user-content-caching-packages-data

https://github.com/actions/cache/blob/main/examples.md#node---npm
Hm. One should not cache the `node_modules` folder at all. So I will have to change this.

**Comment by BenediktSeidl** [comment]
We will need to wait for #65 to see if this works.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (BenediktSeidl) [reply .github/workflows/ci.yml]: I simplified the github action. Thanks for bringing it to my attention!

**Review by PhilippSiegmantel — APPROVED** [review]

**Pull request merged**